docs/architecture/blazor-for-web-forms-developers/intro.md

```diff
@@ -8,6 +8,50 @@ ms.date: 09/11/2019
 
 # An introduction to Blazor for ASP.NET Web Forms developers
 
-ASP.NET Web Forms has been a staple of .NET web development since the .NET Framework first shipped in 2002. 
+ASP.NET Web Forms has been a staple of .NET web development since the .NET Framework first shipped in 2002. Back when the Web was still largely in its infancy, ASP.NET Web Forms made building web apps simple and productive by adopting many of the patterns used for desktop development. In ASP.NET Web Forms apps web pages are composed from reusable UI controls. User interactions are handled as events. A rich ecosystem of Web Forms UI controls provided by Microsoft and third party control vendors makes connecting to data and displaying rich visualizations easy. For the visually inclined, the Web Forms designer provides a simple drag-and-drop interface for laying out controls.
+
+Over the years Microsoft has introduced new ASP.NET based web frameworks to address new trends in web development: ASP.NET MVC, ASP.NET Web Pages, and more recently ASP.NET Core. With each new framework, some have predicted the imminent decline of ASP.NET Web Forms and criticized it as an outdated and outmoded web framework. Despite these predictions, many .NET web developers continue to find ASP.NET Web Forms as a simple, stable, and productive way to get their work done. At the time of this writing almost half a million web developers use ASP.NET Web Forms every month. ASP.NET Web Forms is stable to the point that docs, samples, books, and blog posts from a decade ago remain useful and relevant. For many .NET web developers "ASP.NET" is still virtually synonymous with "ASP.NET Web Forms" as it was when .NET was first conceived. Arguments may rage on the pros and cons of ASP.NET Web Forms relative to the other new .NET web frameworks but ASP.NET Web Forms development remains a popular way to write web apps.
```

Comment:
Do we want to have the doomsaying? Not that it's not true that people say this, but it almost feels like you're agreeing with that sentiment.

Reply:
I was actually trying to side with the Web Forms developers against the doomsayers in this paragraph. The sentiment is supposed to be that despite what the hecklers say, Web Forms continues to be a product way to write web apps used by many people. Any thoughts on on how to change the text to make that clearer?

Reply:
After rereading it (maybe with some updates), I'm not getting that sense anymore. LGTM